docs/statistics.md

```diff
@@ -314,266 +420,230 @@ either already acknowledged or dropped by TSBPD as too late to be delivered.
+TODO: 
 
-The bandwidth is estimated at the receiver.
-The estimation is based on the time between two probing DATA packets.
-Every 16th data packet is sent immediately after the previous data packet.
-By measuring the delay between probe packets on arrival,
-it is possible to estimate the maximum available transmission rate,
-which is interpreted as the bandwidth of the link.
-The receiver then sends back a running average calculation to the sender with an ACK message.
+- What about stats on sender and rcv? smoothed average?
+- there is no receiving speed stats, see also mbpsRecvRate; make a note here
 
-## byteAvailSndBuf
+The receiver records the inter-arrival time of each packet (time delta with the previous data packet) which is further used in the models to estimate bandwidth and receiving speed. The communication between receiver and sender happens by means of acknowledgment packets which are sent regularly (each 10 milliseconds) and contain some control information as well as bandwidth and delivery rate estimations. At the sender side, upon receiving a new value, a smoothed average is used to update the latest estimation mantained at the sender side.
 
-The available space in the sender's buffer, in bytes. Sender only.
+It is important to note that for bandwidth estimation only data probing packets are taken into account while all the data packets (both data and data probing) are used for receiving speed estimation. The idea behind packet pair techniques is to send the groups of back-to-back packets, i.e., probing packet pairs, to a server thus making it possible to measure the minimum interval in receiving the consecutive packets.
 
-This value decreases with data scheduled for sending by the application, and increases 
-with every ACK received from the receiver, after the packets are sent over 
-the UDP link.
+### byteAvailSndBuf
 
-## byteAvailRcvBuf
+The available space in the SRT sender buffer, in bytes. Sender only.
 
-The available space in the receiver's buffer, in bytes. Receiver only.
+TODO: 
 
-This value increases after the application extracts the data from the socket
-(uses one of `srt_recv*` functions) and decreases with every packet received
-from the sender over the UDP link.
+- increases first, decreases second or good?
+- after the packets are sent over the UDP link - ?
 
-## mbpsMaxBW
+This value decreases with the data scheduled for sending by the application and increases with every ACK received from the SRT receiver after the packets are sent over the UDP link.
 
-Transmission bandwidth limit, in Mbps. Sender only.
-Usually this is the setting from 
-the `SRTO_MAXBW` option, which may include the value 0 (unlimited). Under certain 
-conditions a nonzero value might be be provided by a congestion 
-control module, although none of the built-in congestion control modules 
-currently use it.
+### byteAvailRcvBuf
+
+The available space in the SRT receiver buffer, in bytes. Receiver only.
+
+TODO: SRT socket?
```

Comment:
UDP socket is not regarded anyhow in the stats anyway, but might be that this would be better for clarity.